lib/extend/renderer.ts

```diff
@@ -80,10 +41,50 @@ class Renderer {
     return renderer ? renderer.output : '';
   }
 
+  /**
+   * register renderer engine
+   * - [hexo-renderer-nunjucks example](https://github.com/hexojs/hexo-renderer-nunjucks/blob/c71a1979535c47c3949ff6bf3a85691641841e12/lib/renderer.js#L55-L56)
+   * - [typescript example](https://github.com/dimaslanjaka/hexo-renderers/blob/feed801e90920bea8a5e7000b275b912e4ef6c43/src/renderer-sass.ts#L37-L38)
+   * @param name input extension name. ex: ejs
+   * @param output output extension name. ex: html
+   * @param fn renderer function
+   */
```

Comment:
Is it possible to reduce/dedupe the comment here?

Reply:
> Is it possible to reduce/dedupe the comment here?

yes, or should we using shortlink from http://bit.ly to reduce comment length ?